features/FEATURE_LWIP/lwip-interface/mbed_lib.json

```diff
@@ -16,7 +16,11 @@
         "addr-timeout": {
```

Comment:
"addr-timeout" is the amount of time we wait for preferred system address in case we have only non-preferred system address available.
"both-addr-timeout" is the time to wait that both systems addresses are available. 

So the flags have different meaning and both are valid.